corehq/apps/accounting/tasks.py

```diff
@@ -159,12 +159,25 @@ def warn_subscriptions_not_active(based_on_date=None):
         log_accounting_error("%s is not active" % subscription)
 
 
+def warn_active_subscriptions_per_domain_not_one():
+    for domain_name in Domain.get_all_names():
+        active_subscription_count = Subscription.objects.filter(
+            subscriber__domain=domain_name,
+            is_active=True,
+        ).count()
+        if active_subscription_count > 1:
+            log_accounting_error("Multiple active subscriptions found for domain %s" % domain_name)
+        elif active_subscription_count == 0:
+            log_accounting_error("There is no active subscription for domain %s" % domain_name)
```

Comment:
is this going to generate a lot of emails every day? if so might be worth making it a soft assert instead? no strong feelings either way, just something to think about